internal/sdk/cloudian/sdk.go

```diff
@@ -112,6 +112,30 @@ type SecurityInfo struct {
 	SecretKey string `json:"secretKey"`
 }
 
+// QoS is the Cloudian API's term for limits on quotas, counts and rates enforced on a `User`
+type QoS struct {
+	// Max storage quota
+	StorageQuota ByteSize
+	// Warning limit storage quota
+	StorageQuotaWarning ByteSize
+	// Max storage quota in number of objects
+	StorageQuotaCount uint64
+	// Warning limit storage quota in number of objects
+	StorageQuotaCountWarning uint64
+	// Max nr of HTTP requests per minute
+	RequestRatePrMin uint64
+	// Warning limit nr of HTTP requests per minute
+	RequestRatePrMinWarning uint64
+	// Max inbound datarate in ByteSize per minute
+	DataRatePrMinInbound ByteSize
+	// Warning limit inbound datarate in ByteSize per minute
+	DataRatePrMinInboundWarning ByteSize
+	// Max outbound datarate in ByteSize per minute
+	DataRatePrMinOutbound ByteSize
+	// Warning limit outbound datarate in ByteSize per minute
+	DataRatePrMinOutboundWarning ByteSize
```

Comment:
It's long already, I think we can allow the `e` in `Per`

```suggestion
	DataRatePerMinOutboundWarning ByteSize
```

```suggestion
	OutboundDataRatePerMinWarning ByteSize
```